### DISCUSSION TITLE
Enable Dataflow Runner v2

### DISCUSSION THREAD

**Discussion by meirf** [flyteorg/flyte#2609] [Q&A]
Dataflow docs have [these instructions](https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#dataflow-runner-v2) for java:
> To enable Runner v2, run your job with the following flag: --experiments=use_runner_v2.

Do you know how to set this in flyte?

**Comment by brucearctor** [comment]
Are you currently running dataflow from flyte [ and without the flags ]?  

Didn't think that there was a formal integration built yet, ex: https://docs.flyte.org/projects/cookbook/en/stable/integrations.html
